src/modules/removeCompleted.js

```diff
@@ -0,0 +1,10 @@
+const removeCompleted = () => {
+  const labels = document.querySelectorAll('label');
+  labels.forEach((label) => {
+    if (label.classList.contains('checked')) {
+      label.parentNode.parentNode.remove();
+    }
+  });
+};
+
+export default removeCompleted;
```

Comment:
According to the project requirements: 

![image](https://user-images.githubusercontent.com/57421823/154471056-c12df927-fd6b-4231-8a57-84811ec11644.png)

Therefore:

- [ ] Please, kindly use `Array.filter()` to delete your completed tasks with the "Clear all completed task" button.

That way, we can meet this particular requirement 💪

Reply:
Done